ast/format_test.go

```diff
@@ -24,13 +24,21 @@ func (ts *testAstFormatSuite) TestAstFormat(c *C) {
 		output string
 	}{
 		// Literals.
+		{`null`, `NULL`},
+		{`true`, `TRUE`},
 		{`350`, `350`},
 		{`345.678`, `345.678`},
-		{`1e-12`, `1e-12`},
+		{`1e-12`, `0.000000000001`},
 		{`null`, `NULL`},
 		{`"Hello, world"`, `"Hello, world"`},
 		{`'Hello, world'`, `"Hello, world"`},
 		{`'Hello, "world"'`, `"Hello, \"world\""`},
+		{`_utf8'你好'`, `"你好"`},
+		{`x'bcde'`, "\xbc\xde"},
```

Comment:
I think we should make the output a binary literal, the output doesn't look right.